modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/internal/type/test/EmailNotificationTypeTest.java

```diff
@@ -160,14 +226,49 @@ public void testSendNotification() throws Exception {
 			user2.getFirstName(),
 			notificationRecipientSettingsMap.get("fromName"));
 		Assert.assertEquals(
-			user2.getEmailAddress(),
+			expectedSingleRecipient,
+			notificationRecipientSettingsMap.get("singleRecipient"));
+
```

Comment:
Email is not sent in the same order every time. The block below is just to cover this scenario.

Reply:
So, in this case, the Assert:
```java
Assert.assertEquals(
			Arrays.toString(actualEmailAddresses),
			expectedEmailAddresses.length, actualEmailAddresses.length);
```
is it not enough?

Reply:
I would let both Asserts in the test because one is testing the property (true/false) and the other is checking the size of the recipients. What do you think?